discord/utils.py

```diff
@@ -79,6 +79,17 @@
 DISCORD_EPOCH = 1420070400000
 
 
+class _MissingSentinel:
+    def __eq__(self, other):
+        return False
+
+    def __repr__(self):
+        return 'MISSING'
```

Comment:
```suggestion
        return '...'
```

Might as well use this since it's less noisy.